README.md

```diff
@@ -62,6 +62,90 @@ poetry run pytest tests/test_dataset.py
 poetry run pytest -m "not integration"
 ```
 
+## Pydantic Models
```

Comment:
@ardila @phil-scale somehow forgot to push this. Could you take a look at this as well?